### ISSUE TITLE
[Ontology] dgProductRadioactive:dgRaTypeCode value is "I=-White". Should be "I-White"

### ISSUE THREAD

**Issue by Waitak** [IATA-Cargo/ONE-Record#180]
The comments refer to "`I-White`" as the accepted value, but the restriction mandates the (non-standard) value of "`I=-White"`.
```
###  https://onerecord.iata.org/DgProductRadioactive#dgRaTypeCode
dgProductRadioactive:dgRaTypeCode rdf:type owl:DatatypeProperty ;
                                  rdfs:domain :DgProductRadioactive ;
                                  rdfs:range [ rdf:type rdfs:Datatype ;
                                               owl:oneOf [ rdf:type rdf:List ;
                                                           rdf:first "I=-White" ;
                                                           rdf:rest [ rdf:type rdf:List ;
                                                                      rdf:first "II-Yellow" ;
                                                                      rdf:rest [ rdf:type rdf:List ;
                                                                                 rdf:first "III-Yellow" ;
                                                                                 rdf:rest rdf:nil
                                                                               ]
                                                                    ]
                                                         ]
                                             ] ;
                                  rdfs:comment "The category of the package or all packed in one. Complete text to be transmitted: I-White, II-Yellow, III-Yellow instead of I, II, III"@en ;
                                  rdfs:label "dgProductRadioactive:dgRaTypeCode"@en .

```

**Comment by lambertciata** [comment]
Thanks, will be modified in next release

**Comment by lambertciata** [comment]
fixed with 3.0.0
